src/quotamodel.c

```diff
@@ -184,12 +183,28 @@ struct LocalRejectMapEntry
 };
 
 /* using hash table to support incremental update the table size entry.*/
-static HTAB *table_size_map = NULL;
+static HTAB        *table_size_map                      = NULL;
+static TimestampTz *table_size_map_last_overflow_report = 0;
```

Comment:
Why did you change the type of the variable to a pointer?

Reply:
because it is now stored in shared memory. I forgot to change 0 to NULL.

Reply:
I mean why

> it is now stored in shared memory

?

Reply:
because otherwise its value is not saved for dynamic workers, which leads to spam warnings. especially in the case of tests.

Reply:
may be add some comments to code, why in one place tz is pointer while in another - not?